mfusepy.py

```diff
@@ -648,9 +648,10 @@ class c_flock_t(ctypes.Structure):  # type: ignore
 _fuse_int32 = ctypes.c_int32 if (fuse_version_major, fuse_version_minor) >= (3, 17) else ctypes.c_int
 _fuse_uint32 = ctypes.c_uint32 if (fuse_version_major, fuse_version_minor) >= (3, 17) else ctypes.c_uint
 if fuse_version_major == 2:
+    _fh_old_type = ctypes.c_ulong if _system != 'OpenBSD' else ctypes.c_uint
```

Comment:
Ideally, this would be reversed to avoid negatives.

`_fh_old_type = ctypes.c_uint if _system == 'OpenBSD' else ctypes.c_ulong`